.github/workflows/test-coverage.yml

```diff
@@ -80,17 +93,85 @@ jobs:
           done
           echo "$(go env GOPATH)/bin" >> "$GITHUB_PATH"
           go install github.com/wadey/gocovmerge@latest
-          covs=$(find $modules -maxdepth 1 -type f -name coverage.out -print)
+          # shellcheck disable=SC2086
```

Comment:
We could safely attend the SC2086 requirement here. I think it would be a good idea actually.